apps/web/src/design-system/button/Button.styles.ts

```diff
@@ -66,7 +66,7 @@ export const getSubtleStyles = (theme) => {
           backgroundColor: dark ? theme.white : 'transparent',
           backgroundClip: 'none !important',
           color: 'transparent',
-          '-webkit-text-fill-color': 'initial !important',
+          WebkitTextFill: 'initial !important',
```

Comment:
@LetItRock I think it should be: `WebkitTextFillColor` ?

Reply:
is there anything i change @scopsy @LetItRock

Reply:
Hi @sagnik3788 if the error that is shown in the issue is not showing anymore I see this as the solution to the issue and nothing else needed to change 😄